pkg/arvo/sys/vane/ames.hoon

```diff
@@ -1301,25 +1301,27 @@
       event-core
     ::
     =/  =open-packet  (decode-open-packet packet our life.ames-state)
-    ::  store comet as peer in our state
+    ::  add comet as an %alien if we haven't already
+    ::
+    =?  peers.ames-state  ?=(~ ship-state)
+      (~(put by peers.ames-state) sndr.packet %alien *alien-agenda)
+    ::  upgrade comet to %known via on-publ-full
+    ::
+    =.  event-core
```

Comment:
tbh I'm not sure I understand the distinction between `event-core` and `..event-core` here. How can I check which one is correct?